I wish_If only/sofia_zahoruiko.md

```diff
@@ -0,0 +1,29 @@
+1. Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+if only you had come earlier to assess all inferiority of architecture proposed by a first-year student
+
+2. Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+if only we waited for his arrival, we would keep abreast on what stage is the project now and what we should do with it next
+
+3. Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+i wish we had met before leaving and discuss the reason of your dismissal, i have a lot of thoughts in this case and i want to share it with you
+
+4. Жаль что проект завершился так и не начавшись ввиду нехватки кадров и соответствующего финансирования. Если бы все сложилось то задумка вполне могла выстрелить на рынке и стать новым хитом.
+i wish project had more people and appropriate funding to not ends as soon as it starts. If everything gone well, then the idea could well shoot in the market and become a new hit.
+
+5. Он пожалел, что пришел и привел с собой друга. На его месте я бы так не поступал. Если бы он пришел вовремя, этого могло и не случиться.
+He regretted he have come and take a friend with him. in his stead i didnt do this. if only he had come in time it might not have happened.
```

Comment:
If only he hadn't come and brought his frind with him. If I were him I wouldn't have acted like this. If he had come on time, it could have not happened.